dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -521,6 +535,7 @@ private void resolve(Object installObject) {
             }
             for (List<RepositoryResource> item : resolveResult) {
                 for (RepositoryResource repoResrc : item) {
+                    featuresResolved.add(repoResrc.getId());
```

Comment:
If the resource is from the directory based repository, are its Maven coordinates null?  If so, should we return either Maven coordinates or the ID, but not both?  Otherwise each item would have two entries in the result, which doesn't cause any issues but is confusing.

Reply:
When it is a directoryBasedRepository the ID is a relative path (regular ids from the maven repo esas are not paths), and Maven coordinates are null for directoryBasedRepository IDs. There are no duplicates and any nonconforming ids/nulls are removed after it passes through downloadEsas() in the plugin.